contrib/msggen/msggen/schema.json

```diff
@@ -23330,7 +23330,7 @@
           ]
         },
         "amount": {
-          "type": "sat",
+          "type": "msat",
```

Comment:
Are we sure this is `msat`? It is a method that has on-chain impact, and therefore cannot actually use anything sub-1sat, and has to round `msat` values to `sat` for it to be useful. I'll check the docs, but this goes agaisnt my intuition.

Reply:
Nope the docs say:

> amount is the satoshi value that we will contribute to the channel.
This value will be added to the provided PSBT in the output which is
encumbered by the 2-of-2 script for this channel.

So this change needs to be undone.

Reply:
are you sure? all the other onchain commands use msat stuff aswell e.g. withdraw:
```json
"satoshi": {
        "type": "msat_or_all",
        "description": [
          "The amount to be withdrawn from the internal wallet (expressed, as name suggests, in satoshi). The string *all* can be used to specify withdrawal of all available funds (but if we have any anchor channels, this will always leave at least `min-emergency-msat` as change). Otherwise, it is in satoshi precision; it can be a whole number, a whole number ending in *sat*, a whole number ending in *000msat*, or a number with 1 to 8 decimal places ending in *btc*."
        ]
      },
```

Reply:
`sat` is currently not used by any active method (i throws an error even iirc)

Reply:
Oh, ok, that's my oversight :thinking: 

Sorry about that. We should likely backfill that with a real `sat` and `sat_or_all` type, but that can be done in a separate PR.

Reply:
We tried to get rid of all "sat" inputs, but we missed.  The schemas are *NOT* correct for some (e.g. utxopsbt).  It doesn't matter if you always use a suffix (*YOU SHOULD DO THIS*) of course.

There are too many now (esp DF code), so we will have to bless "sat" and "sat_or_all" in the schemas :(